src/create_disk.sh

```diff
@@ -94,12 +94,12 @@ esac
 
 bootfs=$(getconfig "bootfs")
 grub_script=$(getconfig "grub-script")
-ostree=$(getconfig "ostree-repo")
+ostree_container=$(getconfig "ostree-container")
```

Comment:
Another angle to look at this from is that we are dropping another dependency in the code on an unpacked ostree repository, moving to where "source of truth" is a container image (that has an ostree commit inside still).